### PR TITLE
socok8s: Add PR testing job

### PR DESCRIPTION
**Pull request by toabctl**
We want to test pull requests against github so add a job that uses a
Jenkinsfile.integration (see [0]) for testing.

This can be expanded to also add jobs for doc-build, linting, ...

[0] https://github.com/SUSE-Cloud/socok8s/pull/329

### CODE REVIEW

**Comment by toabctl** [comment]
> Can we move the periodic jobs into that too?
> I like this new setup.

yes. on the TODO. But I want to fix PR testing first.

**Comment by Itxaka** [comment]
do we have to create a change per job that we want to add or is it possible to regex the jenkinsfile.* files so they appear automatically once we add them into the socok8s repo?

File: jenkins/ci.suse.de/cloud-socok8s.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+- project:
+    name: cloud-socok8s-jobs/socok8s
```
Comment (stefannica) [inline jenkins/ci.suse.de/cloud-socok8s.yaml]: Nice ! I didn't know you could use folders to group jobs like this.

**Comment by evrardjp** [comment]
> Could you amend your commit message to remove the reference to [0] [SUSE-Cloud/socok8s#329](https://github.com/SUSE-Cloud/socok8s/pull/329) ?

Nevermind. The commit message is correct, the PR text has been adapted to change the PR from 328 to 329.

**Comment by toabctl** [comment]
> do we have to create a change per job that we want to add or is it possible to regex the jenkinsfile.* files so they appear automatically once we add them into the socok8s repo?

Currently we need a change per job. I think that's what we want to have different jobs for eg intergration testing, doc builds, linters, ...

**Comment by evrardjp** [comment]
> > do we have to create a change per job that we want to add or is it possible to regex the jenkinsfile.* files so they appear automatically once we add them into the socok8s repo?
> 
> Currently we need a change per job. I think that's what we want to have different jobs for eg intergration testing, doc builds, linters, ...

I prefer like that anyway, we are whitelisting jobs we know are fine.

**Pull request merged**